lib/bolt/inventory/options.rb

```diff
@@ -124,6 +125,12 @@ module Options
           description: "A map of variables for the group or target.",
           type: Hash,
           _plugin: true
+        },
+        "version" => {
+          description: "The version of the inventoryfile.",
```

Comment:
```suggestion
          description: "The version of the inventory file.",
```

Docs have standardized on two separate words for 'inventory file'.